diff -Naur enigma2-PLi/lib/dvb/epgcache.cpp enigma2-PersianPrince/lib/dvb/epgcache.cpp
--- enigma2-PLi/lib/dvb/epgcache.cpp	2018-07-08 10:43:54.000000000 +0430
+++ enigma2-PersianPrince/lib/dvb/epgcache.cpp	2018-07-08 10:45:02.000000000 +0430
@@ -30,7 +30,7 @@
 /* Interval between "garbage collect" cycles */
 #define CLEAN_INTERVAL 60000    //  1 min
 /* Restart EPG data capture */
-#define UPDATE_INTERVAL 3600000  // 60 min
+#define UPDATE_INTERVAL 60000 // 1 min
 /* Time to wait after tuning in before EPG data capturing starts */
 #define ZAP_DELAY 2000          // 2 sec
 
